src/main/java/me/jellysquid/mods/sodium/client/util/workarounds/DriverWorkarounds.java

```diff
@@ -13,6 +13,10 @@ public static void beforeContextCreation() {
         if (Workarounds.isWorkaroundEnabled(Workarounds.Reference.NVIDIA_BAD_DRIVER_SETTINGS)) {
             NVIDIAWorkarounds.install();
         }
+
+        if (Workarounds.isWorkaroundEnabled(Workarounds.Reference.NVIDIA_BAD_DRIVER_LINUX)) {
```

Comment:
This should just use the existing `NVIDIA_BAD_DRIVER_SETTINGS` option and remove the check for Linux.